tests/unit/src/test/resources/semanticdb-scala3/example/ColorEnum.scala

```diff
@@ -0,0 +1,6 @@
+package example
+
+enum Color/*example.Color#*/(val rgb/*example.Color#rgb.*/: Int/*scala.Int#*/):
+   case Red/*example.Color.Red.*/   extends Color/*example.Color#*/(0xFF0000)/*scala.runtime.EnumValue#*/
```

Comment:
https://github.com/lampepfl/dotty/issues/11689

Reply:
What do you think about adding a link to this issue on a line comment in `ColorEnum.scala`, so we can see what's wrong with the test later?

Reply:
The problem is that it will show up for any other tests too. I will put it into the test suite itself.